packages/utils/scss/typography/_vertical-align.scss

```diff
@@ -1,4 +1,38 @@
-// TODO: docs
+/// This is equivalent to `text-align: left;`.
```

Comment:
text-align: left is not vertical, but horizontal align, the whole file needs to be changed with vertical alignment